2021-11-25 13:57:07,828 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-25 15:24:06,702 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-25 15:24:21,536 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-26 08:02:32,071 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-26 08:18:54,119 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-26 12:27:27,064 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-26 12:49:31,672 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-26 13:04:55,842 INFO: Microblog startup [in C:\Users\Maestro\Desktop\microblog\app\__init__.py:46]
2021-11-26 13:04:57,980 ERROR: Exception on / [GET] [in C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py:1457]
Traceback (most recent call last):
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Maestro\Desktop\microblog\app\routes.py", line 35, in index
    return render_template('index.html', title='Home', form=form,
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\loaders.py", line 137, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 757, in compile
    self.handle_exception(source=source_hint)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\Maestro\Desktop\microblog\app\templates\index.html", line 23, in template
    {% if prev_url %}
jinja2.exceptions.TemplateSyntaxError: expected token 'end of statement block', got '{'
2021-11-26 13:05:16,288 ERROR: Exception on /index [GET] [in C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py:1457]
Traceback (most recent call last):
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Maestro\Desktop\microblog\app\routes.py", line 35, in index
    return render_template('index.html', title='Home', form=form,
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\loaders.py", line 137, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 757, in compile
    self.handle_exception(source=source_hint)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\Maestro\Desktop\microblog\app\templates\index.html", line 23, in template
    {% if prev_url %}
jinja2.exceptions.TemplateSyntaxError: expected token 'end of statement block', got '{'
2021-11-26 13:05:33,653 ERROR: Exception on /explore [GET] [in C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py:1457]
Traceback (most recent call last):
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Maestro\Desktop\microblog\app\routes.py", line 150, in explore
    return render_template('index.html', title='Explore', posts=posts.items,
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\flask\templating.py", line 148, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\loaders.py", line 137, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 757, in compile
    self.handle_exception(source=source_hint)
  File "C:\Users\Maestro\Desktop\microblog\venv\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\Maestro\Desktop\microblog\app\templates\index.html", line 23, in template
    {% if prev_url %}
jinja2.exceptions.TemplateSyntaxError: expected token 'end of statement block', got '{'
